RecoLocalTracker/SiPixelRecHits/src/PixelCPEClusterRepair.cc

```diff
@@ -0,0 +1,628 @@
+// Include our own header first
+#include "RecoLocalTracker/SiPixelRecHits/interface/PixelCPEClusterRepair.h"
+
+// Geometry services
+#include "Geometry/TrackerGeometryBuilder/interface/PixelGeomDetUnit.h"
+#include "Geometry/TrackerGeometryBuilder/interface/RectangularPixelTopology.h"
+
+//#define DEBUG
+
+// MessageLogger
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+
+// Magnetic field
+#include "MagneticField/Engine/interface/MagneticField.h"
+
+
+// Commented for now (3/10/17) until we figure out how to resuscitate 2D template splitter
+/// #include "RecoLocalTracker/SiPixelRecHits/interface/SiPixelTemplateSplit.h"
+
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+
+#include <vector>
+#include "boost/multi_array.hpp"
+
+#include <iostream>
+
+using namespace SiPixelTemplateReco;
+//using namespace SiPixelTemplateSplit;
+using namespace std;
+
+namespace {
+   constexpr float micronsToCm = 1.0e-4;
+   constexpr int cluster_matrix_size_x = 13;
+   constexpr int cluster_matrix_size_y = 21;
+}
+
+//-----------------------------------------------------------------------------
+//  Constructor.
+//
+//-----------------------------------------------------------------------------
+PixelCPEClusterRepair::PixelCPEClusterRepair(edm::ParameterSet const & conf,
+                                           const MagneticField * mag,
+                                           const TrackerGeometry& geom,
+                                           const TrackerTopology& ttopo,
+                                           const SiPixelLorentzAngle * lorentzAngle,
+					   const SiPixelTemplateDBObject * templateDBobject,
+					   const SiPixel2DTemplateDBObject * templateDBobject2D )
+: PixelCPEBase(conf, mag, geom, ttopo, lorentzAngle, nullptr, templateDBobject, nullptr,1)
+{
+   //cout << endl;
```

Comment:
Are these commented out lines needed? If so, please add another line of comment which explain why; otherwise just remove them

Reply:
OK, if you require this change now.